File: 30-days-of-code/day6-lets-review/lets-review.py
#!/bin/python3

if __name__ == '__main__':
    n = int(input())

    for i in range(0, n):
        word = input()

        odd = ""
        even = ""

        for j in range(0, len(word)):
            if j % 2 == 0:
                odd = odd + word[j]
            else:
                even = even + word[j]
            
        print(odd + " " + even)

File: 30-days-of-code/day6-lets-review/lets-review.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>


int main() {
    int n;
    std::cin >> n;

    for (int i {0}; i < n; i++) {
        std::string word, even, odd;
        std::cin >> word;

        for (int j {0}; j < word.length(); j++) {
            if (j % 2 == 0)
                odd += word[j];
            else
                even += word[j];
        }

        std::cout << odd << " " << even << std::endl; 

    }

    return 0;
}
